pkg/gadget-service/api/api.proto

```diff
@@ -96,4 +173,5 @@ service GadgetManager {
   rpc GetInfo(InfoRequest) returns (InfoResponse) {}
   rpc GetGadgetInfo(GetGadgetInfoRequest) returns (GetGadgetInfoResponse) {}
   rpc RunGadget(stream GadgetControlRequest) returns (stream GadgetEvent) {}
+  rpc RunOCIGadget(stream GadgetControlRequest) returns (stream GadgetEvent) {}
```

Comment:
Why is the argument `GadgetControlRequest` implemented as a stream? Why run several gadgets in the same `RunOCIGadget` method call?

Reply:
We don't allow running several gadgets in one request - but we do allow control messages afterwards (this has been like that before also).

So after opening the stream, the first message is always the RunRequest - afterwards a `Stop` can be sent from the client (after pressing CTRL+C for example) and still make sure the gadget can do proper cleanup (and result-sending).
In the future this could also allow us to send other control messages (like changing gadget params while the gadget is running or something).